documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -112,7 +114,11 @@ You can migrate virtual machines to {virt} from the command line.
 
 [IMPORTANT]
```

Comment:
@sbeskin-redhat This entire admonition is just a list Prerequisites. Why is it in an admonition?

Reply:
@stoobie  It's not my idea. Probably to better draw the user's attention to it. As Maayan explained to me, users tend to skip important prerequisites and then have problems with migration and ask for assistance.